core/src/test/java/org/infinispan/api/flags/FlagsEnabledTest.java

```diff
@@ -127,9 +128,19 @@ public void testReplicateSkipCacheLoad(Method m) {
       final String v = v(m, 1);
       final String k = k(m, 1);
       cache1.withFlags(Flag.SKIP_CACHE_LOAD).put(k, v);
-      // The write-skew check tries to load it from persistence.
-      assertLoadsAndReset(cache1, isTxCache() ? 1 : 0, cache2, 0);
 
+      Address primaryLocation = cache1.getDistributionManager().getPrimaryLocation(k);
```

Comment:
In general I prefer using MagicKeys, so that the primary owner is always the same.